patches/net/minecraft/client/data/models/ModelProvider.java.patch

```diff
@@ -0,0 +1,140 @@
+--- a/net/minecraft/client/data/models/ModelProvider.java
++++ b/net/minecraft/client/data/models/ModelProvider.java
+@@ -34,20 +_,58 @@
+     private final PackOutput.PathProvider blockStatePathProvider;
+     private final PackOutput.PathProvider itemInfoPathProvider;
+     private final PackOutput.PathProvider modelPathProvider;
++    public final String modId;
+ 
++    // NeoForge: Use the below constructor passing in your mod id
```

Comment:
Convention is for comments in patches to use `Neo:`. Also, reword the comment to avoid using position, and instead refer to the additional parameter.
```suggestion
+    // Neo: Use the constructor which accepts a mod ID.
```